src/pages/constitution/ConstitutionTOC.tsx

```diff
@@ -1,31 +1,40 @@
 import React from 'react';
 import {
   IonContent,
-  IonItem,
   IonList,
   IonListHeader,
   IonLabel,
   IonPage,
-  withIonLifeCycle
+  withIonLifeCycle,
+  IonIcon,
+  IonButton
 } from '@ionic/react';
+import {document } from 'ionicons/icons';
 import './Constitution.css';
 import { TOCList } from '../../components/constitutionTOC';
+import { ActionButton } from '../../components/actionButton';
 
 class ConstitutionTOC extends React.Component {
   render() {
     return (
       <IonPage>
         <IonContent>
-          <IonListHeader>
-            <IonLabel>Constitution of the Republic of South Africa</IonLabel>
-          </IonListHeader>
-          <IonList>
-            <IonItem routerLink={"/constitution/full"}>Read the full Constitution</IonItem>
-          </IonList>
+          <div className="ion-padding">
+            <section className="tab-title ion-justify-content-between">
```

Comment:
I suggest doing this as three elements, rather than two: the icon, the h2, and the button. Then set the h2 to `flex-grow-1` (and drop ion-justify-content-between) so that it expands to take up space. That way if the h2 wraps, it doesn't go under the icon.